invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -211,10 +206,7 @@
           "description": "TODO - Bucket id - what about third party storage systems?",
           "type": "string"
         },
-        "file_id": {
```

Comment:
I would leave this as ``file_id`` becuase it's not the primary identifier for the file. It's the identifier for the FileInstance, which points to the file on disk (this one can be changed, hence not a primary identifier).